overrides/scripts/BlastFurnace.zs

```diff
@@ -15,6 +15,7 @@ furnace.remove(<appliedenergistics2:material:5>);
 furnace.remove(<thermalfoundation:material:167>, <gregtech:meta_item_1:2708>);
 furnace.remove(<thermalfoundation:material:166>, <gregtech:meta_item_1:2706>);
 furnace.remove(<thermalfoundation:material:165>, <gregtech:meta_item_1:2707>);
+furnace.remove(<ore:ingotOsmium>);

```

Comment:
Are we not the ones adding the smelting recipe? If so, we should remove where we add it, rather than adding it to remove it again.